core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -6191,4 +6191,19 @@ class KafkaApisTest {
     val response = verifyNoThrottling[ConsumerGroupHeartbeatResponse](requestChannelRequest)
     assertEquals(Errors.GROUP_AUTHORIZATION_FAILED.code, response.data.errorCode)
   }
+
+  @Test
+  def testConsumerGroupDescribeReturnsUnsupportedVersion(): Unit = {
+    // Arrange
+    val consumerGroupDescribeRequestData = new ConsumerGroupDescribeRequestData()
+    consumerGroupDescribeRequestData.groupIds.add("group0")
+    val requestChannelRequest = buildRequest(new ConsumerGroupDescribeRequest.Builder(consumerGroupDescribeRequestData, true).build())
+
+    // Act
+    createKafkaApis().handle(requestChannelRequest, RequestLocal.NoCaching)
+    val response = verifyNoThrottling[ConsumerGroupDescribeResponse](requestChannelRequest)
+
+    // Assert
+    assertEquals(Errors.UNSUPPORTED_VERSION.code, response.data.groups.get(0).errorCode)
```

Comment:
nit: In this case, we usually prefer to assert the entire response to ensure that it only contain what we expect. For instance, we would build the ConsumerGroupDescribeResponseData and use assertEquals(expected, response). We can keep it as it is now but let's change it when we update this test during the implementation of the next part.

Reply:
I just changed it now, so I won't forget